{"uuid": "63b629e3-dc1a-4c9e-9c0f-572161e9639b", "children": ["591286bb-d9bb-47b4-b9d8-d18b92196e3d"], "befores": [{"name": "browser", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"/home/max/venv/lib/python3.6/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/max/venv/lib/python3.6/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/max/venv/lib/python3.6/site-packages/_pytest/fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/max/logging_in_PO/conftest.py\", line 81, in browser\n    desired_capabilities=caps)\n  File \"/home/max/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py\", line 157, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/home/max/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py\", line 252, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n  File \"/home/max/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py\", line 319, in execute\n    response = self.command_executor.execute(driver_command, params)\n  File \"/home/max/venv/lib/python3.6/site-packages/selenium/webdriver/remote/remote_connection.py\", line 374, in execute\n    return self._request(command_info[0], url, body=data)\n  File \"/home/max/venv/lib/python3.6/site-packages/selenium/webdriver/remote/remote_connection.py\", line 402, in _request\n    resp = http.request(method, url, body=body, headers=headers)\n  File \"/home/max/venv/lib/python3.6/site-packages/urllib3/request.py\", line 79, in request\n    method, url, fields=fields, headers=headers, **urlopen_kw\n  File \"/home/max/venv/lib/python3.6/site-packages/urllib3/request.py\", line 170, in request_encode_body\n    return self.urlopen(method, url, **extra_kw)\n  File \"/home/max/venv/lib/python3.6/site-packages/urllib3/poolmanager.py\", line 375, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n  File \"/home/max/venv/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 706, in urlopen\n    chunked=chunked,\n  File \"/home/max/venv/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 445, in _make_request\n    six.raise_from(e, None)\n  File \"<string>\", line 3, in raise_from\n  File \"/home/max/venv/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 440, in _make_request\n    httplib_response = conn.getresponse()\n  File \"/usr/lib/python3.6/http/client.py\", line 1373, in getresponse\n    response.begin()\n  File \"/usr/lib/python3.6/http/client.py\", line 311, in begin\n    version, status, reason = self._read_status()\n  File \"/usr/lib/python3.6/http/client.py\", line 272, in _read_status\n    line = str(self.fp.readline(_MAXLINE + 1), \"iso-8859-1\")\n  File \"/usr/lib/python3.6/socket.py\", line 586, in readinto\n    return self._sock.recv_into(b)\n"}, "start": 1632399082228, "stop": 1632399095982}], "start": 1632399082228, "stop": 1632399100168}